.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,17 @@
 <!--
     Don't forget!
-    
+
+    - Sign your commits and our DCO:  https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits
```

Comment:
We should not require GPG signing of git commits. That sets the bar too high.

It's better to have a relevant DCO link. Also, it is not "our" DCO.